### ISSUE TITLE
USWDS-Site - Refresh resources linked on Design Principles page

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds-site#2573]
### Is your feature request related to a problem? Please describe.

Resources linked throughout the [Design Principles page](https://designsystem.digital.gov/design-principles/#start-with-real-user-needs) are aging or no longer relevant. 

### Describe the solution you'd like

We should review links on the page to find fresher content. 

### Additional context

[Links audit doc](https://docs.google.com/spreadsheets/d/1n2qzP7F2osR-kis2wSJcwZsXZ69WUMzQzxvVsMnc7PA)
[Page updates doc](https://docs.google.com/document/d/1gJRt8o78gftS6Ur7p-1UYVynzEw7OLqda-RiZu6CCn8)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate feature requests.

**Comment by annepetersen** [comment]
We will need to include this issue on the TTS audit until it’s resolved

**Comment by finekatie** [comment]
Created additional issue to audit the links: [Audit links on Design principles page #2733](https://github.com/uswds/uswds-site/issues/2733)

**Comment by finekatie** [comment]
The audit and recommendations were made here: [Design principles page _link audit ](https://docs.google.com/spreadsheets/d/1n2qzP7F2osR-kis2wSJcwZsXZ69WUMzQzxvVsMnc7PA)

**Comment by finekatie** [comment]
Reviewed Link audit. [Filtered by what recommendations and comments were made](https://docs.google.com/spreadsheets/d/1n2qzP7F2osR-kis2wSJcwZsXZ69WUMzQzxvVsMnc7PA) on outdated or redundant links. Needs review.

**Comment by finekatie** [comment]
Does this need to be with Jacline for now?

**Comment by finekatie** [comment]
Jacline has reviewed and made edits and recommendations. Handed off to me today and I checked in with Anne. Anne to take one final pass and perhaps share with Dan. Then I will create PRs to address correcting old links and revising others. I think we might need sub tasks for this. Not sure.

**Comment by annepetersen** [comment]
Went quickly through and completed fed column, but I'm waiting on some calls in there. I've made comments to reflect in the spreadsheet

**Comment by annepetersen** [comment]
Other than lines 54 and 57, my review is now complete. @amycole501 should be able to make decisions on this in my absence.

**Comment by jaclinec** [comment]
I took another look and added comments to line 54 and added content to line 57 in the "UX comments" column.

**Comment by finekatie** [comment]
[Created document](https://docs.google.com/document/d/1gJRt8o78gftS6Ur7p-1UYVynzEw7OLqda-RiZu6CCn8) to start making changes

**Comment by finekatie** [comment]
Draft done. Asked for some clarification on questions on spreadsheet. When done, doc can be reviewed and finalized pre-PR. Reminder: We need to add changelog to the page when we update.

**Comment by finekatie** [comment]
Met with Dan today for CS/Comms review. He'd like to get us moving on this. He approved leaving the link as is so I don't need Anne's review again. We'll have more review when the page us updated in the pr.

**Comment by annepetersen** [comment]
@finekatie We actually have to go back through these links given https://github.com/uswds/uswds-site/pull/3068 since the performance links will all 404 now

**Comment by finekatie** [comment]
Ok @annepetersen , I added myself back on this issue and moved it back to In Progress. When you get done making notes in the draft, I'll update in the PR and resubmit for review. Sound good? cc: @jaclinec

**Comment by annepetersen** [comment]
Re-review done

**Comment by finekatie** [comment]
@annepetersen and @thisisdano, links are updated, team review is complete. This is ready for FFR.

**Comment by annepetersen** [comment]
I think this is [blocked by this at the moment](https://github.com/uswds/uswds-site/pull/3053#issuecomment-2669897351)

**Comment by finekatie** [comment]
All subtasks are complete as of today with completion of https://github.com/uswds/uswds-site/pull/3053. This issue is done.
